interpreter/cling/include/cling/std.modulemap

```diff
@@ -369,6 +370,7 @@ module "std" [system] {
     header "codecvt"
   }
   module "cuchar" {
+    requires cplusplus17
```

Comment:
Are you sure about this? `libstdc++` quotes the header as part of C++11.

Reply:
`cuchar` seems unavailable in gcc-5: https://godbolt.org/z/faxaExcr3

I think it exposes the problem we discussed earlier where we can have gcc with c++14/17 building using libstdc++ < 6.

Reply:
As before, I would argue that a compiler not providing `cuchar` is not compliant with C++11 (while ROOT now requires at least C++14), but if it's not a problem for this case, I'm ok with overapproximating.

Reply:
I think it should be fine because we are not making the 'canonical' setup less performant (in terms of memory) but some weird platform setups which will go away over time (when getting rid of gcc-5 support).